code/modules/clothing/modular_armor/attachments.dm

```diff
@@ -45,6 +45,8 @@
 
 	///Layer for the attachment to be applied to.
 	var/attachment_layer
+	///Slot that is required for the action to appear to the equipper. If null the action will appear whenever the item is equiped to a slot.
+	var/prefered_slot = SLOT_WEAR_SUIT
```

Comment:
preferred

https://dictionary.cambridge.org/dictionary/english/preferred

Reply:
Also wouldnt something like 
slot to show for
be better since its not a preference but a requirement